README.md

```diff
@@ -172,9 +161,6 @@ To understand when the lifecycle methods are called take a look at the following
 | ``pause``       | ``onPause``    | ``onPause``        |
 | ``destroy``     | ``onDestroy``  | ``onDestroy``      |
 
-####Renderers
-
-Finally, Rosie includes the [Renderers] [renderers] library to simplify your ``RecyclerView`` handling code. If you decide to use Renderers, remember to extend directly from ``RosieRenderer<T>`` to have ButterKnife injections for free in your renderer views.
```

Comment:
Remember to remove the links related to ``Renderers`` and ``ButterKnife`` at the end of the file.